awx/ui_next/src/screens/Credential/shared/CredentialForm.jsx

```diff
@@ -103,6 +108,7 @@ function CredentialFormFields({
             <Title size="md">{i18n._(t`Type Details`)}</Title>
             {
               {
+                [gceCredentialTypeId]: <GoogleComputeEngineSubForm />,
```

Comment:
on Line 105 you are checking `formik.values.credential_type`.  You could just get the value off of credential_type field(using useField) above on line 30.

Reply:
Awesome - good call.  I think it's definitely cleaner to access the value via the hook.